src/app/home-page/home-logged-in/home-logged-in.component.html

```diff
@@ -13,10 +13,148 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-
 <div fxLayout>
   <app-my-sidebar fxFlexAlign="stretch"></app-my-sidebar>
 
+  <div id="row-holder" fxLayout="column" fxLayoutAlign="start start" class="p-3 w-100">
+    <mat-card class="alert alert-info mx-1 w-100" role="alert"> Major Release: </mat-card>
+    <div id="entry-holder" fxLayout="row" fxLayoutGap="10px" class="w-100 mb-4">
+      <mat-card class="mat-card-grow">
+        <div id="header-bar" fxLayout="row" fxLayoutGap="5px" class="mat-card-header-separate">
+          <div id="workflow-number" fxLayout="row">
+            <mat-card-header fxLayout="row">
+              <mat-card-title>Workflows</mat-card-title>
+              <span class="bubble">0</span>
+            </mat-card-header>
+          </div>
+          <button
+            mat-button
+            class="private-btn small-btn-structure ml-3 mb-3"
+            id="registerWorkflowButton"
+            matTooltip="Register Workflow"
+            matTooltipPosition="after"
+          >
+            <img src="../../../assets/svg/icons-plus.svg" alt="Register a tool" class="pr-2 pb-1" />
+            Register a Workflow
+          </button>
+        </div>
+        <mat-divider></mat-divider>
+        <mat-card-content class="p-3">
+          <mat-form-field appearance="outline" class="search-form-field w-100">
+            <mat-label>Search your Workflows...</mat-label>
```

Comment:
In the case of `0` workflows (or tools/services), the [zeplin mockup](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/6022ab2d42df9f2c57c20982) doesn't show the search bar and recent activity in the component. It also shows a "You have not registered any ___" message. (Not sure if this will be added later)

Reply:
You're right, this will be added later! I just didn't have a condition to work with yet :)